azure/packages/azure-local-service/package.json

```diff
@@ -23,15 +23,19 @@
     "clean": "rimraf dist lib *.tsbuildinfo *.build.log",
     "eslint": "eslint --format stylish src",
     "eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
+    "format": "npm run prettier:fix",
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
+    "prettier": "prettier --check . --ignore-path ../../../.prettierignore",
+    "prettier:fix": "prettier --write . --ignore-path ../../../.prettierignore",
     "start": "forever start dist/index.js",
     "start:debug": "node --inspect=0.0.0.0:9229 dist/index.js",
     "stop": "forever stop dist/index.js",
     "test": "echo \"Error: no test specified\" && exit 1",
     "tsc": "tsc"
   },
   "dependencies": {
+    "prettier": "~2.6.2",
```

Comment:
Ditto

Reply:
I added `prettier` in `azure-local-service` and `test/scenario-runner` packages. 

Rest of the other packages already had `prettier` with them. For example, I was already able to run `npm format` to run prettier in `azure-client` packages. 

Prettier is also included as a `devDependencies` inside the `azure-client/package.json`, thus I thought it was okay to apply the existing pattern to other packages within the other azure packages.

Reply:
You say that `azure-client` has it in `devDependencies`, not `dependencies`, correct? My point is to do the same here. The two packages in this PR have it under `dependencies`.